canonical-web-and-design/tools.demos.haus
# Add notice about Firefox bug

## The problem

Firefox does not write svg to canvas unless the root svg element has the width and height specified

## Done

- Add width and height to Ubuntu logo so it can be displayed in Firefox
- Add notice informing users about Firefox bug with rendering SVG to canvas

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8057/?title=Ubuntu+for+the+internet+of+things&subtitle=From+smart+homes+to+smart+drones%2C+robots%2C+and+industrial+systems%2C+Ubuntu+is+the+new+standard+for+embedded+Linux.&illustration_url=https%3A%2F%2Fassets.ubuntu.com%2Fv1%2F049b4cc6-ubuntu%2Bcore%2B18%2Bcircular%2Bwhite.svg%3Fw%3D250&background=dark
- Check that the Ubuntu logo is present on all banners in Firefox
- Check that there is a notice informing users that the illustrations are not rendered in Firefox
